holoviews/element/selection.py

```diff
@@ -83,12 +83,13 @@ def spatial_select_gridded(xvals, yvals, geometry):
 def spatial_select_columnar(xvals, yvals, geometry, geom_method=None):
     if 'cudf' in sys.modules:
         import cudf
+        import cupy as cp
         if isinstance(xvals, cudf.Series):
             xvals = xvals.values.astype('float')
             yvals = yvals.values.astype('float')
             try:
                 import cuspatial
-                result = cuspatial.point_in_polygon(
+                result = cuspatial.point_in_polygon(  # TODO: now only takes two arguments
```

Comment:
@AjayThorve, can you help me get `cuspatial` up and running if you have time?

I get this error `TypeError: point_in_polygon() takes 2 positional arguments but 6 were given` when running the test. I assume this has worked at some point.

Reply:
Hey @Hoxbro. yes there have been some changes in the point_in_polygon api, it only takes two parameters now, both cuspatial GeoSeries (points, polygons). Here is something that might work:

```python
import cudf
import cupy as cp
from shapely.geometry import Polygon
import geopandas

if isinstance(xvals, cudf.Series):
    xvals = xvals.values.astype('float')
    yvals = yvals.values.astype('float')
    try:
        import cuspatial
        df = cudf.DataFrame({'x':xvals, 'y':yvals})
        points = cuspatial.GeoSeries.from_points_xy(
           df.interleave_columns().astype('float')
        )
        polygons = cuspatial.GeoSeries(
           geopandas.GeoSeries(Polygon(geometry)), index=["selection"]
        )

        result = cuspatial.point_in_polygon(points.polygon)

        return result.selection
```

ref: 
1. https://docs.rapids.ai/api/cuspatial/nightly/api_docs/geopandas_compatibility/#cuspatial.GeoSeries

2. https://docs.rapids.ai/api/cuspatial/nightly/user_guide/cuspatial_api_examples/#cuspatial.point_in_polygon

Reply:
Thank you so much! Worked like a charm.

Do you have a version of when the changes were made?